### ISSUE TITLE
chatrr help list `encrypted` attribute instead of `encrypt`

### ISSUE THREAD

**Issue by 3v1n0** [twpayne/chezmoi#772]
```
chattr -h                   
Description:
  Change the attributes of *targets*. *attributes* specifies which attributes to
  modify. Add attributes by specifying them or their abbreviations directly,
  optionally prefixed with a plus sign (`+`). Remove attributes by prefixing
  them or their attributes with the string `no` or a minus sign (`-`). The
  available attributes and their abbreviations are:

    ATTRIBUTE  | ABBREVIATION
  -------------+---------------
    empty      | e
    encrypted  | none
    exact      | none
    executable | x
    private    | p
    template   | t
```

```
chezmoi chattr encrypted ~/foo
chezmoi: encrypted: unknown attribute
```
As the correct one is just `encrypt`

**Comment by twpayne** [comment]
Agreed. This is already implemented in the v2 branch.

**Comment by twpayne** [comment]
Oops, I misunderstood this bug report. Sorry about that. This should be fixed with #833.
